src/framework/components/scroll-view/system.js

```diff
@@ -50,6 +55,12 @@ class ScrollViewComponentSystem extends ComponentSystem {
         if (data.dragThreshold === undefined) {
             data.dragThreshold = DEFAULT_DRAG_THRESHOLD;
         }
+        if (data.useMouseWheel === undefined) {
+            data.useMouseWheel = true;
+        }
+        if (data.mouseWheelSensitivity === undefined) {
+            data.mouseWheelSensitivity = SCROLLBAR_MOUSE_WHEEL_SENSITIVITY;
```

Comment:
Got a slight concern here. So if `data.mouseWheelSensitivity` is set to the _reference_ to `Vec2.ONE`, if `data.mouseWheelSensitivity` with any other value, the actual value of the `Vec2.ONE` constant will change. Might be safer to do the following further up:

```
const SCROLLBAR_MOUSE_WHEEL_SENSITIVITY = new pc.Vec2(1, 1);
```

Reply:
Or make mouseWheelSensitivity a new pc.Vec2 and copy the constant rather than referencing it?

Reply:
Ahh good callout! Will send another commit for this.